build/build.go

```diff
@@ -41,7 +41,7 @@ func NewBuildContext(installSuffix string, buildTags []string) *build.Context {
 		GOARCH:        "js",
 		InstallSuffix: installSuffix,
 		Compiler:      "gc",
-		BuildTags:     append(buildTags, "netgo"),
+		BuildTags:     append(buildTags, "netgo", "purego"),
```

Comment:
Just saw this and I realized I have no idea what `netgo` is. Perhaps separate these out onto each of their own line and comment with a reference to what they are and/or why they are set?

Reply:
Sure, I can document them.

"netgo" is a build tag used in `net` package to get a Go (rather than cgo) version of the `net` package, using Go's DNS resolver rather than a cgo DNS resolver. It's documented at https://godoc.org/net#hdr-Name_Resolution, and you can also grep https://gotools.org/net for "netgo" to see it in action.

It was added to GopherJS in commit 273a70496de06c38fddef4a8b2dd31d0446dcb51.

Reply:
Done in e9ae517; PTAL.

Reply:
LGTM